src/settings.js

```diff
@@ -2048,6 +2048,11 @@ var LOAD_SOURCE_MAP = false;
 // List of path substitutions to apply in the "sources" field of the source map.
 // Corresponds to the ``--prefix`` option used in ``tools/wasm-sourcemap.py``.
 // Must be used with ``-gsource-map``.
+//
+// This setting allows to map path prefixes to the proper ones so that the final
+// (possibly relative) URLs point to the correct locations :
+// ``-sSOURCE_MAP_PREFIXES=['/old/path=/new/path']``
```

Comment:
Can you remove the square brackets and the quotes here?   I'm trying to encourage the simpler list form which is just a comma separated string.